docs/Project.toml

```diff
@@ -1,2 +1,5 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[compat]
```

Comment:
is that part necessary? This will create unnecessary conflicts with https://github.com/JuliaSparse/SparseArrays.jl/pull/444

Reply:
This is necessary to get the docs to build until the upgrade to Documenter v1 is complete

Reply:
I just merged #444.